### PR TITLE
Describe why and how to not support request_uri

### LINKED ISSUE #114: Disallow request_uri in OpenID authentication requests with automatic registration
> Due to the large size of a Trust Chain, it may be necessary to use the HTTP POST method, request_uri, or a [Pushed Authorization Request](https://openid.net/specs/openid-federation-1_0.html#RFC9126) [[RFC9126](https://openid.net/specs/openid-federation-1_0.html#RFC9126)] for the request.

The `request_uri` asks the OP to fetch an external resource and this makes it easier to launch DoS attacks against Federation OPs. I propose to say `request_uri` MUST NOT or SHOULD NOT be used by RPs and accepted by OPs.

OIDC recognised this issue and because of that it has the OP `request_uri_parameter_supported` + `require_request_uri_registration` metadata and the RP `request_uris` metadata. So that the `request_uri`'s can be screened before the RP can use them.

In automatic registration if a `request_uri` is used the OP needs to trust it blindly. If the downloaded JWT contains no Trust Chain what should the OP do then? Stop? Or resolve the `client_id`, fetching more resources?

It's good security policy to limit and minimise the number of resources at unverified URLs that need to be fetched. POSTing the `request` object or PAR is sufficient when dealing with large authentication requests (including requests that carry a `trust_chain`).

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #114

### CODE REVIEW

File: openid-federation-1_0.xml [hunk 1]
```diff
@@ -7081,6 +7090,13 @@ HTTP/1.1 302 Found
           the Federation Entity Discovery process should not be the default
           action for the resolve endpoint in this case.
         </t>
+	<t>
+	  Passing request objects by reference using the
+	  <spanx style="verb">request_uri</spanx> request parameter) parameter
```
Comment (vdzhuvinov) [inline openid-federation-1_0.xml:7095]: Repeating parameter.

**Review by vdzhuvinov — CHANGES_REQUESTED** [review]
All good, just a typo that needs fixing. Thanks!

**Review by rohe — APPROVED** [review]
Provided @vdzhuvinov change request is honoured.

File: openid-federation-1_0.xml (hunk 1 above)
Comment (selfissued) [reply openid-federation-1_0.xml]: Corrected

**Review by vdzhuvinov — APPROVED** [review]

**Comment by selfissued** [comment]
On the 30-Oct-24 Federation editors' call, @ve7jtb suggested the possibility of using HTTP to limit the size of resources received.

Mike said that these OP metadata parameters may be pertinent:
> request_uri_parameter_supported
> OPTIONAL. Boolean value specifying whether the OP supports use of the request_uri parameter, with true indicating support. If omitted, the default value is true.
> require_request_uri_registration
> OPTIONAL. Boolean value specifying whether the OP requires any request_uri values used to be pre-registered using the request_uris registration parameter. Pre-registration is REQUIRED when the value is true. If omitted, the default value is false.

**Comment by selfissued** [comment]
During the 30-Oct-24 Federation editors' call, John was skeptical of removing an OpenID Connect feature.  He suggested ways to mitigate the denial-of-service problems instead, including at the HTTP layer.

**Review by peppelinux — APPROVED** [review]

**Pull request merged**